go/libraries/doltcore/sqle/database.go

```diff
@@ -915,10 +1046,40 @@ func (db Database) dropFragFromSchemasTable(ctx *sql.Context, fragType, name str
 }
 
 // TableEditSession returns the TableEditSession for this database from the given context.
-func (db Database) TableEditSession(ctx *sql.Context) *editor.TableEditSession {
+func (db Database) TableEditSession(ctx *sql.Context, isTemporary bool) *editor.TableEditSession {
+	if isTemporary {
+		return DSessFromSess(ctx.Session).tempTableEditSessions[db.name]
+	}
 	return DSessFromSess(ctx.Session).editSessions[db.name]
 }
 
+// GetAllTemporaryTables returns all temporary tables
+func (db Database) GetAllTemporaryTables(ctx *sql.Context) ([]sql.Table, error) {
```

Comment:
Delete as discussed in gms PR

Reply:
Resolved that this will continue to exist